README.md

```diff
@@ -114,15 +173,18 @@ BENCHMARK(BM_StringCopy);
 BENCHMARK_MAIN();
 ```
 
-Don't forget to inform your linker to add benchmark library e.g. through 
-`-lbenchmark` compilation flag. Alternatively, you may leave out the 
-`BENCHMARK_MAIN();` at the end of the source file and link against 
-`-lbenchmark_main` to get the same default behavior.
+To run the benchmark, compile and link against the `libbenchmark.a` library
```

Comment:
Again, too specific. I'd avoid spelling out the actual filenames.

Reply:
Will update.